Notifications/notifications_actions_customization.md

```diff
@@ -12,99 +12,111 @@ This document is a starting point for engaging the community and standards bodie
 
 ```javascript
-enum NotificationActionStyle {
+enum NotificationScenario {
   "default",
-  "success",
-  "critical"
+  "incoming-call"
 }
 
-dictionary NotificationAction {
-  required DOMString action;
-  required DOMString title;
-  USVString icon;
-  NotificationActionStyle style = "default";
-};
+dictionary NotificationOptions {
+  NotificationScenario scenario = "default";
+  ...
+}
 ```
 
-Similarly, to support this scenario we also propose the creation of a new NotificationSoundType enum and the inclusion of a `sound` attribute to the Notification interface and a correspondent property to the NotificationOptions dictionary:
+When the scenario is `"incoming-call"`, the notification will always have a default dismiss button. This way, when a calling notification is displayed to the user, we can guarantee that he or she will always be capable to dismiss it. 
 
-```javascript
-enum NotificationSoundType {
-  "default",
-  "ringtone"
-}
+Moreover, notifications created within the `"incoming-call"` scenario should have increased priority over the regular ones. That is, if a `"default"` notification is currently being displayed and another `"incoming-call"` notification gets created, it should break through and be displayed immediately, requiring immediate action from the user.
 
-partial interface Notification {
-  readonly attribute NotificationSoundType sound;
-}
+### Color Treatment
 
-dictionary NotificationOptions {
-  ...
-  NotificationSoundType sound = "default";
+For notifications with `"scenario"` of type `"incoming-call"`, the User Agent (UA) may style the notification's action buttons' differently than the `"default"` scenario. Additionally, it is preferable that the default dismiss button is colored distinctly than the others in the `Notfication.actions` array. We propose that the default dismiss action button should have a red theme - e.g., red background color with some predefined icon on it - while all the other buttons are green-themed, but it is up to the platform to define the style.
+
+Finally, if the platform does not allow color treatment for notification buttons, the UA should fallback to the `"default"` scenario colors.
+
+### Ringtone
+
+We also propose that, when a notification is displayed in the `"incoming-call"` scenario, a ringtone is played. The UA should play a predefined - possibly platform-provided - audio, suitable for executing in a loop, and keep playing it for the duration of the notification. If the platform does not allow  displaying a notification with a ringtone, the UA must fallback to the `"default"` scenario sound.
+
+### Use Examples
+
+An incoming call notification from a VoIP PWA without any specified action buttons could be instantiated through a service worker by using:
+
+```javascript
+const title = "Andrew Bares";
+const options = {
+  scenario: "calling",
```

Comment:
Shouldn't it be "incoming-call" instead of "calling"?

Reply:
Yes. Fixed it and other occurrences as well. Thanks!